### ISSUE TITLE
discord development

### ISSUE THREAD

**Issue by Heckerrr** [BetterDiscord/BetterDiscord#1630]
### Before Requesting

- [X] I found no existing issue matching my feature request

### Describe the feature you'd like!

There is a discord app version called "discord development" (its basically discord canary), and if i want to install betterdiscord on it, it simply wont work.

So my feature request is:
can you pls add a option to install betterdiscord for discord development?

### Anything else?

the youtube video where i found discord development:
https://youtu.be/9A4pqljCpOU?t=431

**Comment by rauenzi** [comment]
See above

**Comment by elderlabs** [comment]
Yeah, the Discord Devlopment client is no secret, but there's no reason to use it, either. It's entirely Canary to the standard end-user. Regardless, if you want BD installed with it, pick Canary in the installer list, then change the install location to `%localappdata%\DiscordDevelopment\app-####\modules\discord_desktop_core-1\discord_desktop_core`. It works entirely fine.

If it were me, I'd have a button in the installer that appears if it detects the Development folder, but if you've come _this_ far, you can manually choose your installation folder. 

As for the Youtube link, yeah, the things people do for clicks these days...

If you want my advice, don't use the Development client. There's a reason it's entirely unsupported. Use Canary.

**Comment by garry-ut99** [comment]
> @Heckerrr
> ### Before Requesting
> 
>  * [x]  I found no existing issue matching my feature request


Not true, you didn't even care to search, otherwise you would quickly notice that it's another duplicate of https://github.com/BetterDiscord/BetterDiscord/issues/520, along with the 6 other duplicates: [LINK](https://github.com/BetterDiscord/BetterDiscord/issues?q=is%3Aissue+in%3Atitle+development+sort%3Acreated-desc).

Then you've done the same trick on another repository : `https://github.com/uBlockOrigin/uAssets/issues/19976#issuecomment-1799297070`.

**Comment by rauenzi** [comment]
No need to necro-post...

**Comment by garry-ut99** [comment]
Just like there is no need to create a duplicated spam thread...if OP didn't do it, I wouldn't have had to create a necro-post.., your cherry-picking me out of the 2 and blaming me for the crap OP started, is an inconsistency, because I don't see you writing to OP the same you've written to me: "no need to create a duplicate spam thread"... either lecture the both of us the same way, or none, for the same "crime".

**Comment by rauenzi** [comment]
> if OP didn't do it, I wouldn't have had to create a necro-post

You didn't _have_ to at all.

> because I don't see you writing to OP the same you've written to me

They were told to see #520 and had their issue closed as duplicate...

> Two users did same "crime", but only 1 is being punished.

No one was "punished"?

As for the rest of your unintelligible rant, I truly hope you seek and get the help you need, but this is not the place for such things.

**Comment by garry-ut99** [comment]
> ( https://github.com/BetterDiscord/BetterDiscord/issues/1630#issuecomment-1581310325 ) : Rauezni : See above.

No need to spam-post and create a duplicated comment, if you have nothing new to add, than what was already written in the previous post, at least in my "necro-post" I pointed out an important fact that the guy jumps from a repo to a repo and spams, which is disrespectful towards community, while his duplicate spam-thread and your duplicate spam-comment, introduced nothing than a spam and nothing useful.

> You didn't _have_ to at all.

Just like he didn't have to create his duplicate spam-thread.
And just like you didn't have to create your duplicate spam-post.

> They were told to see #520 and had their issue closed as duplicate...

He was told but not lectured, that's the difference you seem to have troubles realising.

> No one was "punished"?

No need for cheap rethoric arguing, or catching me on typos, see that I've written "crime" in parentheses, 
"punished" was supposed to be written in parentheses too, if you fell better we can even replace it with "lectured".

> As for the rest of your unintelligible rant, 

I don't see anything "unintelligible" in it, just plain arguments, just because you are not smart enought to understand what others say, doesn't necessarily mean I wrote an "unintelligible rant", I think that if you really didn't understand something, you would just ask me in a civilised manner, instead of sticking labels or calling names. You seem to run out of arguments...that's why you are trolling instead, if in a dicsussion you want to deny your opponents's arguments, you have to provide your counter-arguments, a discussion should be based on arguments, not on labels or calling names.

> I truly hope you seek and get the help you need, 

Given a fact that it's you who is out of arguments, and not me, it seems it's you who needs a help, and not me.

> but this is not the place for such things.

If you want to evacuate and end the discussion because you lack arguments to defend, I have nothing against.
I just hope that in all your desperation, you won't use physical arguments instead of real arguments, to eliminate your opponent ( hiding/removing my comments or writing to me and locking the thread / banning me quickly afterwards, so that I can't reply and defend anymore, this is the most common way trolls try to win a discussion: at the moment they run out of real arguments they create their last comment full of crap/manipulation/accusations and then they quickly lock the thread or ban the opponent to prevent him from defending, it's like kicking a person lying down - a very nasty acting).

**Comment by rauenzi** [comment]
I'm not sure if you're just a troll or legitimately angry, but I hope you grow out of this type of behavior. Or at least seek a new venue for it because this certainly isn't the place.
